### DISCUSSION TITLE
动态字段出现问题

### DISCUSSION THREAD

**Discussion by 131452099900** [milvus-io/milvus#30918] [Q&A and General discussion]
我使用的是java的sdk，版本是2.3.2，在开启动态字段后插入不存在的字段，查询返回的是这样一串东西，好像字段并没有插入成功，而且无论我插入什么类型都是这个，是现在java的sdk不支持吗。
fields_data {
  type: JSON
  field_name: "$meta"
  scalars {
    json_data {
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
      data: "{}"
    }
  }
  field_id: 104
  is_dynamic: true
}

**Comment by yhmo** [comment]
客户端代码怎么写的

**Reply by 131452099900** [reply]
客户端代码并没有出错，我再三检查过了字段是有值的，我试了三种数据类型的动态插入都不行，分别是varchar，long，还有json，最终结果都如上所示，我用的是测试数据，动态随机生成的，数据是能插入的只是动态字段无效。

**Comment by yhmo** [comment]
https://github.com/milvus-io/milvus-sdk-java/blob/339387c2017681cc8f1a641d0fded5e7df4c5681/src/test/java/io/milvus/client/MilvusClientDockerTest.java#L1322
这个测试代码是插入动态字段然后query出来的，你对着这个代码看吧。

**Comment by xiaofan-luan** [comment]
please share your code that could reproduce the bug if possible
